code/__DEFINES/traits.dm

```diff
@@ -129,20 +130,27 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_HUSK "husk"
 #define TRAIT_BADDNA "baddna"
 #define TRAIT_CLUMSY "clumsy"
-#define TRAIT_CHUNKYFINGERS "chunkyfingers" //means that you can't use weapons with normal trigger guards.
+/// means that you can't use weapons with normal trigger guards.
+#define TRAIT_CHUNKYFINGERS "chunkyfingers"
 #define TRAIT_DUMB "dumb"
-#define TRAIT_ADVANCEDTOOLUSER "advancedtooluser" //Whether a mob is dexterous enough to use machines and certain items or not.
-#define TRAIT_MONKEYLIKE "monkeylike" //Antagonizes the above.
+/// Whether a mob is dexterous enough to use machines and certain items or not.
+#define TRAIT_ADVANCEDTOOLUSER "advancedtooluser"
+// Antagonizes the above.
```

Comment:
```suggestion
/// Antagonizes the above.
```

Reply:
unless it was intentional, since it references the above

Reply:
It was intentional--this is a joke comment, not a documentative comment.

Reply:
i'm half-convinced it's not, since monkeylike is a pretty vague name, and it originally was commented to clarify it was meant specifically in terms of tool use

that trait should ideally get either a more descriptive name or its own comment to avoid confusion

Reply:
I thought it was a real comment as well...

Reply:
A more descriptive name is created in https://github.com/tgstation/tgstation/pull/57897.